### DISCUSSION TITLE
How to fix "label: unnamed-chunk-X"

### DISCUSSION THREAD

**Discussion by avishai987** [quarto-dev/quarto-cli#2645] [Q&A] [answered]
Hi, I'm trying to give a name to the chunk with #|label: , but the chunk is still called "unnamed-chunk-X" in the console and the figure-html folder. 
How can I fix that? Thanks!

**Comment by dragonstyle** [comment, marked as answer]
The following should be sufficient (note the space between the `|` and the `label`).

```
#| label: fig-airquality
```

**Reply by avishai987** [reply]
Thanks!
